### DISCUSSION TITLE
Trouble launching

### DISCUSSION THREAD

**Discussion by Ruttie2006** [CosmosOS/Cosmos#2557] [Q&A]
Whenever I try to run it, I get the following error:
![image](https://user-images.githubusercontent.com/80394237/213178202-c376558f-329b-45be-b7e2-eff8e8502ffe.png)
Any ideas as to why this is happening?
(Translation for last line: "The system cannot find the given path")

**Comment by AsertCreator** [comment]
did you install vmware? you can use both types. make sure you installed vmware at default path.

**Reply by Ruttie2006** [reply]
I did not change the path, but just to be sure, what is the default path?

**Reply by Ruttie2006** [reply]
(I have `C:\Program Files (x86)\VMware\VMware Player\vmplayer.exe`)
